images/build/Dockerfile

```
@@ -0,0 +1,19 @@
+FROM docker.io/library/golang:1.19.9 as download
+
+WORKDIR /tmp
+ENV KUBECTL_VERSION="v1.26.3"
```

We can use build args to parametrize the Dockerfile. This would help make it extensible in future.

```suggestion
ARG GO_VERSION
ARG K8S_VERSION

FROM docker.io/library/golang:${GO_VERSION} as download

WORKDIR /tmp
```